packages/tooltip/theme/lumo/vaadin-tooltip-styles.js

```diff
@@ -12,8 +12,8 @@ const tooltipOverlay = css`
   }
 
   [part='overlay'] {
-    background-color: var(--lumo-contrast);
-    color: var(--lumo-primary-contrast-color);
+    background: var(--lumo-base-color), linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
```

Comment:
Either remove the comma (one background declaration with both color and image), or move the color after the gradient (two background declarations, first one is visually on top of the second one).

```suggestion
    background: var(--lumo-base-color) linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
```

Reply:
Thanks, removed the comma.